hack/local-up-cluster.sh

```diff
@@ -406,7 +413,7 @@ function start_apiserver {
     fi
 
     # Admission Controllers to invoke prior to persisting objects in cluster
-    ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount${security_admission},ResourceQuota,DefaultStorageClass,DefaultTolerationSeconds
+    ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount${security_admission},ResourceQuota,DefaultStorageClass,DefaultTolerationSeconds${ADMISSION_CONTROL_ADD}
```

Comment:
those should move before ResourceQuota

for a generic insertion point, it really depends. much of what was discussed in https://github.com/kubernetes/kubernetes/pull/34131/files still applies, regarding different phases/purposes of admission (defaulting, overriding, and validating)